src/weakest_precondition/mod.rs

```diff
@@ -160,11 +157,11 @@ pub fn gen(index: usize, data: &mut MirData, post_expr: &Option<Expression>, deb
     wp
 }
 
-/// Returns the type of an operand as a String (ie: "i32", "bool", etc.)
+/// Returns the type of an operand as a `String` (ie: `i32`, `bool`, etc.)
```

Comment:
@salterm If this is the line you're referring to, these aren't single quotes, but backticks, which formats this word to be monospaced in the documentation.